feature/container/containerz/tests/container_lifecycle/README.md

```diff
@@ -98,6 +98,23 @@ Using the container started as part of CNTR-1.2, validate that the container can
 be stopped, and is subsequently no longer listed in the `gnoi.Containerz.List`
 API.
 
+## CNTR-1.5: Restart Behaviour
+
+Using the containers started as part of CNTR-1.1, restart the device via gNOI
+and ensure that the containers are still running post reboot.
+
+## CNTR-1.6: Failover Behaviour
+
+Using the containers started as part of CNTR-1.1, trigger a failover and 
+ensure that the containers that were running on the failed RP are running 
+on the backup RP.
+
+## CNTR-1.7: Syslog logging
+
+Using the containers started as part of CNTR-1.1, configure syslog logging
+such that container logs are exported over syslog. The resulting logs
+should identify which container originated the logs.
```

Comment:
How will this be verified?  This sounds implementation dependent which we should avoid in our functional tests.  We also should avoid using syslog for any test verification.  (syslog should be reserved for manual debug only)

Is there some RPC that can be used to achieve your test goal?

Reply:
Understood - dropped this test